test/common_quantization.py

```diff
@@ -454,20 +455,26 @@ def __init__(self, quantized=False):
         if quantized:
             self.mycat = nnq.QFunctional()
             self.myadd = nnq.QFunctional()
+            self.my_intrinsic_add = nni.quantized.modules.AddReLU()
         else:
             self.mycat = nnq.FloatFunctional()
             self.myadd = nnq.FloatFunctional()
+            self.my_intrinsic_add = nni.AddReLU()
+
             self.mycat.observer = DummyObserver()
             self.myadd.observer = DummyObserver()
+            self.my_intrinsic_add.observer = DummyObserver()
 
     def forward(self, x):
         y = self.mycat.cat([x, x, x])
         z = self.myadd.add(y, y)
+        z = self.my_intrinsic_add(z, z)
         return z
 
     @classmethod
     def from_float(cls, mod):
         new_mod = cls(quantized=True)
         new_mod.mycat = new_mod.mycat.from_float(mod.mycat)
         new_mod.myadd = new_mod.myadd.from_float(mod.myadd)
+        new_mod.my_intrinsic_add = new_mod.my_intrinsic_add.from_float(mod.my_intrinsic_add)
```

Comment:
and why do we write a separate from_float rather than relying on the tool?

Reply:
I don't remember the reason :/